price-estimator/openapi.yml

```diff
@@ -178,3 +182,10 @@ components:
         list:
           summary: Two Addresses
           value: 0x00000000000000000000000000000000000000a0,0x00000000000000000000000000000000000000A1
+    BlockNumber:
+      name: blockNumber
+      in: query
+      description: The block number to compute the estimate for. This will use th open orderbook at that block (i.e. orders that will be considered for solving the current batch at the block number). This parameter cannot be specified together with the "batchId" query parameter.
```

Comment:
```suggestion
      description: The block number to compute the estimate for. This will use the open orderbook at that block (i.e. orders that will be considered for solving the current batch at the block number). This parameter cannot be specified together with the "batchId" query parameter.
```